documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -591,12 +939,12 @@ TBD
 Further Examples
 ----------------
 
-To take a cut from an existing ``sqw`` or ``dnd`` object, retaining the existing
-projection axes and binning:
+To take a cut from an existing ``sqw`` or ``dnd`` object retaining the existing
+projection, omit the ``proj`` argument:
```

Comment:
to be pedantic, it's not omitted, it's blanked with the empty object

Reply:
```suggestion
projection, provide an empty ``proj`` argument:
```